hipchat.go

```diff
@@ -201,9 +204,14 @@ func (c *Client) listen() {
 				continue
 			}
 
+			// parse time
+			time_pieces := strings.Split(attr["ts"], ".")
+			sec, _ := strconv.ParseInt(time_pieces[0], 10, 64)
```

Comment:
thanks so much for the details.

i trust the 'bot' fix will handle my issue, so please feel free to kill
this PR.

cheers

On Thu, Jan 22, 2015 at 3:19 PM, Dane Harrigan <notifications@github.com>
wrote:

> In hipchat.go
> <https://github.com/daneharrigan/hipchat/pull/5#discussion_r23410739>:
>
> > @@ -201,9 +204,14 @@ func (c *Client) listen() {
> >  				continue
> >  			}
> >
> > +			// parse time
> > +			time_pieces := strings.Split(attr["ts"], ".")
> > +			sec, _ := strconv.ParseInt(time_pieces[0], 10, 64)
>
> completely fair questions. the likelihood is next to nil. the error
> handling is just best practice, but hipchat communicates with a tweaked
> version of xmpp. there are some hipchat specific attributes added to the
> xmpp stanza so having seen that its better to code defensively.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/daneharrigan/hipchat/pull/5/files#r23410739>.
>



-- 
taw